docs/pages/empiar_tutorial.md

```diff
@@ -1219,7 +1221,7 @@ These `.pkl` files contain a numpy array of indices into the particle stack, whi
 
 #### (Additional Functionality) Writing a new .star file
 
-- The selected particles can be converted to a .star file using `cryodrgn write_starfile`. See Section 8.3 for more details on this script.
+- The selected particles can be converted to a .star file using `cryodrgn_utils write_star`. See Section 8.3 for more details on this script.
```

Comment:
Not exactly related to issue #151, but `write_starfile` is not yet a `cryodrgn`  command. We can change this if/when it does become one.